src/objects/portal.py

```diff
@@ -0,0 +1,18 @@
+from src.common.exec_interrupt import ExecInterrupt
+from src.common.point import Point
+from .object import Object
+
+
+class Portal(Object):
+    '''
+    `Object` allowing the player to move to a next `scene`.
+    '''
+
+    def __init__(self, scene, position: Point = Point(0, 0)) -> None:
+        super().__init__(position, scene)
+        self.walkable = True
+
+    def on_collision(self, other) -> None:
+        self.scene.runtime.next_scene.execute()
+        # Stop execution flow of `exec`
+        raise ExecInterrupt
```

Comment:
proponuję, aby zmienić nazwę rzucanego wyjątku

Reply:
Na jaką?

Reply:
Czekam na inwencje twórczą xd

Reply:
Are we going to use it for anything other than scene change?
If not, maybe just let's name the exception `SceneChange`.

Reply:
Chosen `MapExit`, because it will always quit a map but not always load next.